docs/platforms/javascript/common/user-feedback/v7/index.mdx

```diff
@@ -0,0 +1,279 @@
+| `inputOutlineFocus`       | `--input-outline-focus`        | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Outline color for form inputs when focused                                                            |
+| `formBorderRadius`        | `--form-border-radius`         | `20px`                                    | `20px`                                    | Border radius style for the form                                                                      |
+| `formContentBorderRadius` | `--form-content-border-radius` | `6px`                                     | `6px`                                     | Border radius style for form content (for example: inputs, buttons)                                   |
+
+Here is an example of customizing only the background color for the light theme using the Feedback constructor configuration:
+
+```javascript
+feedbackIntegration({
+  themeLight: {
+    background: "#cccccc",
+  },
+});
+```
+
+Or the same example above, but using the CSS variables method:
+
+```css
+#sentry-feedback {
+  --background: #cccccc;
+}
+```
+
+### Additional UI Customization
+
+Similar to theme customization above, these are additional CSS variables that can be overridden. These aren't supported in the constructor.
+
+| Variable        | Default                                 | Description                                                                       |
+| --------------- | --------------------------------------- | --------------------------------------------------------------------------------- |
+| `--bottom`      | `1rem`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--right`       | `1rem`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--top`         | `auto`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--left`        | `auto`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--z-index`     | `100000`                                | The z-index of the widget                                                         |
+| `--font-family` | `"'Helvetica Neue', Arial, sans-serif"` | Default font-family to use                                                        |
+| `--font-size`   | `14px`                                  | Font size                                                                         |
+
+### Event Callbacks
+
+Sometimes it’s important to know when a user started interacting with the feedback form, so you can add custom logging, or start/stop background timers on the page until the user is done.
+
```

Comment:
```suggestion
Because it’s sometimes useful to know when a user started interacting with the feedback form, we've made it possible for you to add custom logging, or start and stop background timers on the page that tell you when the user is done.

```